app/models/event.rb

```diff
@@ -1,6 +1,6 @@
 class Event < ApplicationRecord
   belongs_to :place
-  belongs_to :user
```

Comment:
You should not remove this relation. This is relation which tells us who is the owner of event. You should add new relation `has_and_belongs_to_many :users` which will tell us who is going to this event. Probably name `users` is not the best here. We will take care of it soon. This change is also the reason why your user condition stopped working.

Reply:
like that:

class Event < ApplicationRecord
  belongs_to :place
  has_and_belongs_to_many :users
  belongs_to :user

  validates :name, name_lenght: { minimum: 5 }
end
??

Reply:
Yes, but you can group all `belongs_to` to one place.

Reply:
Yes, but you can group all `belongs_to` together.

Reply:
I have tried to do it in one line.... but doesn`t work...

Reply:
No I mean, something like alphabetical order. So you will have `belongs_to` line by line and then the rest.